### PR TITLE
Extend descriptive characteristics

### PR DESCRIPTION
**Pull request by heinpa**

### CODE REVIEW

**Comment by Perevalov** [comment]
@heinpa please check the tests output. also, remove the test for python3.6 from the actions `.yml` file

File: tests/test_descriptive_statistics.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 from linguaf import descriptive_statistics as ds
```
Comment (Perevalov) [inline tests/test_descriptive_statistics.py:1]: Please, check the output of the tests within this PR. Something doesn't work as expected

**Review by Perevalov — CHANGES_REQUESTED** [review]

File: tests/test_descriptive_statistics.py (hunk 1 above)
Comment (heinpa) [reply tests/test_descriptive_statistics.py:1]: for 3/4 of the test this was related to missing language models. I've made changes to address this issue.

File: linguaf/__init__.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import json
 
-SUPPORTED_LANGS = ['en', 'ru']
+SUPPORTED_LANGS = ['en', 'ru', 'de', 'fr', 'es', 'zh', # stopwords from nltk
+                   'lt', 'be', 'uk', 'hy'] # stopwords from other sources
 __version__ = '0.1.0'
```
Comment (Perevalov) [inline linguaf/__init__.py:5]: bump version (patch) https://semver.org/

File: pytest.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[pytest]
```
Comment (Perevalov) [inline pytest.ini:1]: shall we add a tests for German language now?

Comment (heinpa) [reply pytest.ini:1]: Most of the tests have already been extended for German (and the other requested languages), see https://github.com/WSE-research/LinguaF/pull/19/files#diff-ad414dbb73de3f5749cf12f9d21f036319bebbf00443a9c2c23a67fb6edd8ae9R98 for example (you have to load the diff manually). Are you referring to these types of tests, or are you thinking of different ones?

File: .github/workflows/python-package.yml [hunk 4]
```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
```
[the remark this answers is not in the document]
Comment (heinpa) [reply .github/workflows/python-package.yml]: seems like the trailing 0 in python-version is dropped during packaging: https://github.com/WSE-research/LinguaF/actions/runs/8972717045/job/24641323926?pr=19#step:3:7

File: pytest.ini (hunk 3 above)
Comment (Perevalov) [reply pytest.ini:1]: All good, thanks!

**Pull request merged**